packages/manager/modules/pci/src/projects/project/storages/databases/database/namespaces/edit/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,30 @@
+{
+  "pci_databases_namespaces_edit_back": "Namespaces",
+  "pci_databases_namespaces_edit_title": "Modifier un namespace",
+  "pci_databases_namespaces_edit_name_label": "Nom",
+  "pci_databases_namespaces_edit_name_placeholder": "Nom de votre namespace",
+  "pci_databases_namespaces_edit_type_label": "Type",
+  "pci_databases_namespaces_edit_type_popover": "Il ne peut y avoir qu'un seul namespace non aggrégé",
+  "pci_databases_namespaces_edit_retention_perdiod_duration_label": "Durée de rétention",
+  "pci_databases_namespaces_edit_retention_perdiod_duration_placeholder": "Ex: 48h",
+  "pci_databases_namespaces_edit_retention_perdiod_duration_popover": "Durée de rétention de la data pour le namespace en format short time. Exemples: 20s, 10m, 24h, 1d",
```

Comment:
**issue:** Looks like translation key has a typo `perdiod`. Shouldn't be `period`?


```suggestion
  "pci_databases_namespaces_edit_retention_perdiod_duration_popover": "Durée de rétention de la data pour le namespace en format short time. Exemples : 20s, 10m, 24h, 1d",
```

Reply:
Yes, good catch ! Changed: https://github.com/ovh/manager/pull/6414/commits/381dc3d497cbf9f1deff3d720cc87222dc6ef677